### DISCUSSION TITLE
关于vineyard集群环境下多机通信的问题。

### DISCUSSION THREAD

**Discussion by UVUUUVUU** [v6d-io/v6d#820] [General]
您好。想请教一下，已经通过helm在kubernetes下部署了master+2node的vineyard集群，helm部署后提示也已经启动了该服务，并提供了rpc通信端点。但是v6d.io官网有一句话：（Note that when deploying vineyard on Kubernetes, it usually can only be connected from containers in the same pod, or pods on the same hosts.）。我的理解这句话的意思是vineyard客户端只能连接到本pod或本主机的其他pod进行数据共享，那这和k8s的数据挂载有什么区别呢，是单纯的多了内存的零拷贝吗？还有就是既然只能本机pod间通信，那提供的rpc接口的意义在哪里呢，还是说目前只是有这样一个rpc接口，但实际功能并未完善呢？谢谢。

**Comment by dashanji** [comment]
官网的意思是内存零拷贝只能发生在同一台机器/节点上，不同节点上的数据传输用rpc。
